hub-templates/basehub/values.yaml

```diff
@@ -298,8 +298,24 @@ jupyterhub:
         c.JupyterHub.spawner_class = CustomSpawner
 
       06-custom-authenticator: |
+        from z2jh import get_config
         from oauthenticator.auth0 import Auth0OAuthenticator
 
+        staff_names_google = [
+          "yuvipanda@gmail.com",
+          "choldgraf@gmail.com",
+          "georgiana.dolocan@gmail.com",
```

Comment:
Is there a reason we are using personal emails here and not our 2i2c.org addresses? They should work because they're powered by Google, yes? Or am I missing something?